pyvista/examples/planets.py

```diff
@@ -72,20 +64,22 @@ def load_sun(radius=1.0, lat_resolution=50, lon_resolution=100):  # pragma: no c
     --------
     >>> import pyvista as pv
     >>> from pyvista import examples
+    >>> mesh = examples.planets.load_sun()
+    >>> texture = examples.planets.download_sun_surface(texture=True)
```

Comment:
The disparity between planet and texture function name is not great, but I guess inevitable. And in any case users trying to use these helpers will look at the examples and see the relevant texture function.

Reply:
I'm not a huge fan of this `planets` module overall. IMO, it would be better as a standalone `pyvista-planets` package. Otherwise, this seems like a lot of code for examples that only a small minority of users leverage.

I agree the naming here isn't ideal. I do not want this PR to be the place to refine this and recommend opening an issue for follow up work and to discuss striping this module out into a new `pyvista-examples` repo for instance

Reply:
cc @tkoyama010

Reply:
I made https://github.com/pyvista/pyvista/issues/4526 .